airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -134,10 +129,6 @@ def retry_factor(self) -> float:
         """
         return 5
 
-    @property
-    def authenticator(self) -> HttpAuthenticator:
-        return self._authenticator
-
```

Comment:
What's the upgrade path if you were using this property? I see a decent amount of `self.authenticator` usage in the connectors

Reply:
I've sort of answered my question: This is only set if you're using one of the deprecated classes that aren't coming from `AuthBase`. 

It looks l like what the migrated connectors are doing is accessing `_session.auth` directly, which the connectors could do to migrate over (which is still a bit weird as `_session` is a "private" property

Reply:
yeah using the `_session.auth` param is a little off. I'm not overly concerned about it as the new HttpClient is reusable so there shouldn't be a need for copying an authenticator from a stream to another. Next step is to use the client in the declarative module then we'll update the `HttpStream`

Reply:
Sweet, sounds like a good plan!